client/components/tile-grid/README.md

```diff
@@ -0,0 +1,50 @@
+TileGrid (jsx)
+==========
+
+Component used to declare an area for displaying Tiles — it's the main wrapper that takes care of tile alignment and positioning.
+
+#### Props
+
+* `className`: Add your own class to the grid.
+* `isHidden`: Whether the entire grid should be hidden. Default is `false`.
```

Comment:
Why add a prop to hide the grid (using CSS)? If anyone wants to hide this thing, they're supposed to only render it conditionally (using a ternary or whatever), no?

Or, in other words: Why does `TileGrid` deserve more of an `isHidden` prop than other components? 😄

Reply:
Ah, that makes complete sense. Seems like I was following the original code from signup too blindly 😆 

Thanks, addressed in 14200a43edd3117fa73644feea16f2602b4d66dd.